sysdump/sysdump.go

```diff
@@ -1177,15 +1177,15 @@ func (c *Collector) SubmitCniConflistSubtask(ctx context.Context, pods []*corev1
 			if err != nil {
 				return err
 			}
-			cniConfigFileNames := strings.Split(outputStr.String(), "\n")
+			cniConfigFileNames := strings.Split(strings.TrimSpace(outputStr.String()), "\n")
 			for _, cniFileName := range cniConfigFileNames {
 				cniConfigPath := path.Join(c.Options.CNIConfigDirectory, cniFileName)
 				cniConfigFileContent, err := c.Client.ExecInPod(ctx, p.GetNamespace(), p.GetName(), containerName, []string{
 					catCommand,
 					cniConfigPath,
 				})
 				if err != nil {
-					return err
+					return fmt.Errorf("failed to copy CNI config file '%s' from directory '%s': %w", cniFileName, c.Options.CNIConfigDirectory, err)
```

Comment:
Nit: let's use `%q` here for consistency:

```suggestion
					return fmt.Errorf("failed to copy CNI config file %q from directory %q: %w", cniFileName, c.Options.CNIConfigDirectory, err)
```

Reply:
makes sense, i keep forgetting `%q` is a thing. updated ✅